MonoGame.Framework/Audio/SoundEffect.cs

```diff
@@ -262,13 +262,7 @@ public static int GetSampleSizeInBytes(TimeSpan duration, int sampleRate, AudioC
         /// </remarks>
         public bool Play()
         {
-            var inst = GetPooledInstance(false);
-            if (inst == null)
-                return false;
-
-            inst.Play();
-
-            return true;
+            return Play(1f, 0f, 0f);

```

Comment:
This looks to already be handled in `SoundEffectInstancePool.GetInstance(bool)`:

https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework/Audio/SoundEffectInstancePool.cs#L65